eng/native/configurecompiler.cmake

```
@@ -152,6 +153,19 @@ elseif (CLR_CMAKE_HOST_UNIX)
   endif()
 endif(MSVC)
 
+if(CLR_CMAKE_TARGET_APPLE)
+  add_link_options($<$<BOOL:$<TARGET_PROPERTY:INTERPROCEDURAL_OPTIMIZATION>>:LINKER:object_path_lto,lto.o>)
```

I'm not sure why CMake isn't parsing out the `LINKER:` correctly. Let's try passing the argument in a more straightforward manner.

```suggestion
  add_link_options($<$<BOOL:$<TARGET_PROPERTY:INTERPROCEDURAL_OPTIMIZATION>>:-Wl,object_path_lto,lto.o>)
```